### ISSUE TITLE
Add sprites from Pokemon Scarlet and Violet

### ISSUE THREAD

**Issue by WSLUser** [msikma/pokesprite#135]
I'd like to see the sprites added here from the latest 2 Pokemon games to release. Thanks in advance.

**Comment by arodriguez810** [comment]
Will the scarlet and violet Pokémon icons work in this project? I mention it because I couldn't have icons as good as yours just by resizing.

**Comment by arodriguez810** [comment]
@WSLUser, @msikma I hope it can be of some use to you

[Pokemon Scarlet and Violet Normal and Shiny Icons SV Styles.zip](https://github.com/msikma/pokesprite/files/10147667/Pokemon.Scarlet.and.Violet.Normal.and.Shiny.Icons.SV.Styles.zip)

**Comment by arodriguez810** [comment]
![spritesheet](https://user-images.githubusercontent.com/3308921/205918576-c49cbfc8-3196-445b-a4a3-961515345a57.png)
![spritesheet](https://user-images.githubusercontent.com/3308921/205918597-f92ed125-4876-4e0f-aaa3-62e0f9b84686.png)

**Comment by FlitPix** [comment]
An asset dump is available for Scarlet/Violet: https://mega.nz/file/sBcAFI5B#vKFw0txsg14DD3k-HTNyWZH-uLlJfICXZ7kz-kfGRDk

Source: https://twitter.com/mattyoukhana_/status/1593265628080705538

**Comment by itsjavi** [comment]
maybe it's also worth using the sprites from these pixel artists:

https://www.deviantart.com/nolo33lp/art/Pokemon-Scarlet-and-Violet-front-and-icon-sprites-942959329
https://twitter.com/ezerart_/status/1609572784065204224/photo/1

I asked the last artist about it here: https://twitter.com/supereffectiv/status/1609592549072683014?s=20&t=ynPIoenT4K4ezyDGVvLa7Q

**Comment by itsjavi** [comment]
So the Ezerart (Alex) gave permission to use his assets in this project if we give him credits. He said he is finishing them and once they are done he will publish them in original quality on his Deviantart.

I also got contacted by [MBCMechachu](https://twitter.com/MBCMechachu) to let me know about the [National Pokedex Version Delta](https://www.deviantart.com/mbcmechachu/art/National-Pokedex-Icon-Dex-824897934) project, which has all the sprites for Gen 9 already finished. We only need to add a link to this [credits list](https://docs.google.com/spreadsheets/d/1kI_PDXnbghxjN2LBvxA6Pz-QqMYlVGN3Z1EivXOYwNY/edit#gid=0) in the [contributions.md](contributors.md) file.

As you can see the sprites of this big project are created by many different artists. This is just a personal opinion, but it's worth maybe considering that the ones from Ezerart look more consistent for Gen 9 since they are from the same author.

**Comment by MBCMechachu** [comment]
> ![spritesheet](https://user-images.githubusercontent.com/3308921/205918576-c49cbfc8-3196-445b-a4a3-961515345a57.png) ![spritesheet](https://user-images.githubusercontent.com/3308921/205918597-f92ed125-4876-4e0f-aaa3-62e0f9b84686.png)

Hey just a heads up but these sprite here actually are all stolen from this website: https://yakkun.com/sv/zukan/#filter=9
As for Roaming Form Gimmighoul that sprite originates from Deviantart User CarmaNekko.

**Comment by MBCMechachu** [comment]
> So the Ezerart (Alex) gave permission to use his assets in this project if we give him credits. He said he is finishing them and once they are done he will publish them in original quality on his Deviantart.
> 
> I also got contacted by [MBCMechachu](https://twitter.com/MBCMechachu) to let me know about the [National Pokedex Version Delta](https://www.deviantart.com/mbcmechachu/art/National-Pokedex-Icon-Dex-824897934) project, which has all the sprites for Gen 9 already finished. We only need to add a link to this [credits list](https://docs.google.com/spreadsheets/d/1kI_PDXnbghxjN2LBvxA6Pz-QqMYlVGN3Z1EivXOYwNY/edit#gid=0) in the [contributions.md](contributors.md) file.
> 
> As you can see the sprites of this big project are created by many different artists. This is just a personal opinion, but it's worth maybe considering that the ones from Ezerart look more consistent for Gen 9 since they are from the same author.

I kind of overlooked this one earlier but I've already gone through with showing my icon dex to msikma seeing as how I have contact with them over Discord. Along with that I've also gone through with showing them that I do in fact plan to cover [item sprites](https://www.deviantart.com/mbcmechachu/art/Pokemon-Item-Dex-931091190) now that it's been made clear they aren't coming back either.

**Comment by koenigderluegner** [comment]
By now I looked through many icons and so far I think the ones from Ezeart are the best ones lookwise. The Yakkun-Dex, which is the most spread one, is just a pixelated version of the current "drawn" sprites. Im not trying to talk the effort down or something, but I think the custom made ones from Ezeart look far more Gamefreak-like and clean. 

Also @msikma  if you need help compiling/gathering all the data, let me know. Either via GitHub or Discord (Alex#0491) will work, I didn't start because I don't know how much is already done.

**Comment by itsjavi** [comment]
I agree with @koenigderluegner 
and the Yakkun-Dex icon sizes are 32x32, which are very small compared to the Sw/Sh style
but it might be worth to include them in the future as an alternate size

**Comment by MBCMechachu** [comment]
> By now I looked through many icons and so far I think the ones from Ezeart are the best ones lookwise. The Yakkun-Dex, which is the most spread one, is just a pixelated version of the current "drawn" sprites. Im not trying to talk the effort down or something, but I think the custom made ones from Ezeart look far more Gamefreak-like and clean.
> 
> Also @msikma if you need help compiling/gathering all the data, let me know. Either via GitHub or Discord (Alex#0491) will work, I didn't start because I don't know how much is already done.

> I agree with @koenigderluegner and the Yakkun-Dex icon sizes are 32x32, which are very small compared to the Sw/Sh style but it might be worth to include them in the future as an alternate size

Only issue with that would be finding 3DS Styled sprites done for the gen 8 mon. As of now the custom sprites for ones from Legends Arceus vary in quality. On top of that the Pokesprite project iirc has been using Switch Style for fully evolved mons.

**Comment by itsjavi** [comment]
@MBCMechachu right, mixing different height styles for evolved mons might be a problem

**Comment by talwat** [comment]
Is this still being worked on?

**Comment by itsjavi** [comment]
@talwat yes, there are discussions with msikma and he will do it as soon as he gets the time.
we already have the sprites

**Comment by Bakery300** [comment]
I hope I'm not a bother, I was wondering if this project is still being worked on for adding the Gen 9 sprites or if there's any way to assist. I've been using this Github to run a digital table top and my jurry rigged method of adding the new sprites from my collection of icons just crashed and burned. I made an account to see if everything was till active.

**Comment by martimlobao** [comment]
> @WSLUser, @msikma I hope it can be of some use to you
> 
> [Pokemon Scarlet and Violet Normal and Shiny Icons SV Styles.zip](https://github.com/msikma/pokesprite/files/10147667/Pokemon.Scarlet.and.Violet.Normal.and.Shiny.Icons.SV.Styles.zip)

@arodriguez810 What's the source of those icons? Do you have any of the newer ones after #1010 (Iron Leaves)?

**Comment by martimlobao** [comment]
In case anyone is interested, I created a fork with the new sprites from SV: https://github.com/martimlobao/pokesprite

It's a hard fork, meaning that I'm only including a small subset of all the work @msikma has done, and I'm mainly maintaining it for my own use case. Essentially, I'm primarily only interested in a single canonical sprite version for each Pokémon and Pokémon form (as opposed to keeping different versions for e.g. gen 8 and 7). I'll also be adding shiny sprites where I can find them, but only as a secondary goal. I've also kept the item sprites but might delete them in the future.

If this sounds like something anyone is interested in, feel free to use the sprites in your spreadsheet or whatever.

**Comment by MBCMechachu** [comment]
> > @WSLUser, @msikma I hope it can be of some use to you
> > [Pokemon Scarlet and Violet Normal and Shiny Icons SV Styles.zip](https://github.com/msikma/pokesprite/files/10147667/Pokemon.Scarlet.and.Violet.Normal.and.Shiny.Icons.SV.Styles.zip)
> 
> @arodriguez810 What's the source of those icons? Do you have any of the newer ones after #1010 (Iron Leaves)?

If you scroll up you'll see arodriguez810 actually took those sprites from https://yakkun.com/sv/zukan/#filter=9

**Comment by bamq** [comment]
For anyone just wanting icons for the Scarlet & Violet Pokemon, I've gone ahead and used the [National Pokédex Version Delta](https://www.deviantart.com/mbcmechachu/art/National-Pokedex-Icon-Dex-824897934) ones and resized them to match the msikma ones so they look perfect when used together. https://github.com/bamq/pokemon-sprites

**Comment by itsjavi** [comment]
@bamq nice work! thank you for putting them together!

what's the license? may I use them for pokepc.net / supereffective.gg  ?

**Comment by bamq** [comment]
Sure, just give credit to [them](https://www.deviantart.com/mbcmechachu/art/National-Pokedex-Icon-Dex-824897934) and their [artists](https://docs.google.com/spreadsheets/d/1kI_PDXnbghxjN2LBvxA6Pz-QqMYlVGN3Z1EivXOYwNY/edit?gid=0#gid=0).
